wk5/5.a/memorygame.js

```diff
@@ -78,10 +79,11 @@ function mousePressed() {
         if (gameState.flippedCards.length < 2 && cards[k].didHit(mouseX, mouseY)) 
         {
            console.log('flipped', cards[k]);
-           gameState.flippedCards.push(card[k]);
+           gameState.flippedCards.push(cards[k]);
         }
     }
     if (gameState.flippedCards.length === 2) {
+        console.log("2 flipped lets check")
```

Comment:
also just for me -- I wanted to check if the matcher condition was firing and it was just fine